### ISSUE TITLE
Required description on update custom field

### ISSUE THREAD

**Issue by antiuser** [osTicket/osTicket#5856]
Hi there!
I have one custom field (datetime), i want follow function:
If agent change this custom field, then this agent "should" leave a comment for this update.
Is it real?

Thanks a lot!

**Comment by JediKev** [comment]
@antiuser 

That’s already built-in functionality with Inline Edit. When an Agent updates any Inline field there will be an optional text box to fill out a reason why.

Cheers.

**Comment by antiuser** [comment]
@JediKev  Yep, im understand that, but i want to know how to do this optional text required?

**Comment by JediKev** [comment]
@antiuser 

You will have to modify the code to do that.

Cheers.

**Comment by antiuser** [comment]
@JediKev will be glad if you point me what i should edit...

I can explain why we need this function.
One agent modify field (schedule time) and nothing said to other colleagues, then confusing others.
If comment will be required this will avoid this problem
